<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-N84GOOJ" failures="1" tests="13" name="org.TestCases.TC03_DepositsValidation" time="823.473" errors="0" timestamp="2023-04-13T17:19:52 IST" skipped="0">
  <testcase classname="org.TestCases.TC03_DepositsValidation" name="TC01_RegistrationValidation" time="46.011"/>
  <system-out>
    <![CDATA[Registration --> Passed ]]>
  </system-out>
  <testcase classname="org.TestCases.TC03_DepositsValidation" name="TC02_LoginValidation" time="21.134"/>
  <system-out>
    <![CDATA[Login --> Passed ]]>
  </system-out>
  <testcase classname="org.TestCases.TC03_DepositsValidation" name="TC03_SIBDreams" time="80.989"/>
  <system-out>
    <![CDATA[Interest Rate 4.6%
Interest Benefits ` 8.63
Monthly Contribution ` 106.90
Reference ID :2023041303177406
Transaction ID :0940103000000304

SIB Dreams Deposit Account Creation --> Passed ]]>
  </system-out>
  <testcase classname="org.TestCases.TC03_DepositsValidation" name="TC04_FixedDeposit" time="89.946"/>
  <system-out>
    <![CDATA[Reference ID :2023041303177407
Transaction ID :0940100000000517

Fixed Deposit Account Creation --> Passed ]]>
  </system-out>
  <testcase classname="org.TestCases.TC03_DepositsValidation" name="TC05_TaxSaver" time="102.743"/>
  <system-out>
    <![CDATA[Reference ID :2023041303177408
Transaction ID :0940100000000518

Tax Saver Deposit Account Creation --> Passed ]]>
  </system-out>
  <testcase classname="org.TestCases.TC03_DepositsValidation" name="TC06_FlexiDepositOpen" time="89.459"/>
  <system-out>
    <![CDATA[Reference ID :2023041303177410
Transaction ID :0940100000000519

Flexi Deposit Account Creation --> Passed ]]>
  </system-out>
  <testcase classname="org.TestCases.TC03_DepositsValidation" name="TC07_RecurringDepositOpen" time="82.779"/>
  <system-out>
    <![CDATA[Reference ID :2023041303177411
Transaction ID :0940103000000305

Recurring Deposit Account Creation --> Passed ]]>
  </system-out>
  <testcase classname="org.TestCases.TC03_DepositsValidation" name="TC08_DepositDetails" time="40.417"/>
  <system-out>
    <![CDATA[
 Validated - Deposit Calculator -> Fixed Deposit  

 Validated - Deposit Calculator -> Recurring Deposit   

 Validated -> EMI Calculator  
Validating Deposit Details --> Passed ]]>
  </system-out>
  <testcase classname="org.TestCases.TC03_DepositsValidation" name="TC09_LoanAgainstDepositNeg" time="59.685"/>
  <system-out>
    <![CDATA[Loan Against Deposit Account Creation Negative Scenario --> Passed ]]>
  </system-out>
  <testcase classname="org.TestCases.TC03_DepositsValidation" name="TC010_LoanAgainstDepositPos" time="59.704">
    <failure message="The following asserts failed:
	expected [Successful!!] but found [Sorry Try Again!]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Successful!!] but found [Sorry Try Again!]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at org.Pages.DepositsPage.ladvalidation(DepositsPage.java:648)
at org.TestCases.TC03_DepositsValidation.TC010_LoanAgainstDepositPos(TC03_DepositsValidation.java:259)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC010_LoanAgainstDepositPos -->
  <system-out/>
  <testcase classname="org.TestCases.TC03_DepositsValidation" name="TC011_DepositClosureRD" time="63.904"/>
  <system-out>
    <![CDATA[Deposit Closure RD --> Passed ]]>
  </system-out>
  <testcase classname="org.TestCases.TC03_DepositsValidation" name="TC012_DepositClosureFD" time="64.219"/>
  <system-out>
    <![CDATA[Deposit Closure FD --> Passed ]]>
  </system-out>
  <testcase classname="org.TestCases.TC03_DepositsValidation" name="TC_013_LogOff" time="22.483"/>
  <system-out>
    <![CDATA[Logout --> Passed ]]>
  </system-out>
</testsuite> <!-- org.TestCases.TC03_DepositsValidation -->
